.evergreen/config.yml

```diff
@@ -1,1303 +1,61 @@
-  tasks:
-    - name: "test-standalone"
-    - name: "test-standalone-auth"
-    - name: "test-standalone-ssl"
-    - name: "test-replicaset"
-    - name: "test-replicaset-auth"
-    - name: "test-sharded"
-
-# Test alternative storage engines on MongoDB 4.0
-- matrix_name: "test-storage-engines"
-  matrix_spec: { "os": "debian92", "mongodb-versions": "4.0", "php-edge-versions": "latest-stable", "storage-engine": ["inmemory", "mmapv1"] }
-  display_name: "${storage-engine}: ${os}, ${mongodb-edge-versions}, ${php-edge-versions}"
-  tasks:
-    - name: "test-standalone"
-
-- matrix_name: "libmongoc-versions"
-  matrix_spec: { "os": "debian11", "mongodb-edge-versions": "latest-stable", "php-edge-versions": "latest-stable", "libmongoc-version": "*" }
-  display_name: "libmongoc ${libmongoc-version}: ${os}, ${mongodb-edge-versions}, ${php-edge-versions}"
-  tasks:
-    - name: "test-standalone-ssl"
-    - name: "test-replicaset-auth"
-    - name: "test-sharded"
-
-- matrix_name: "atlas-connectivity-tests"
-  matrix_spec: { "os": "debian11", "php-edge-versions": "latest-stable" }
-  display_name: "Atlas Connectivity"
-  tasks:
-    - name: "test-atlas"
-
-- matrix_name: "test-ocsp"
-  matrix_spec: { "os": "debian10", "mongodb-versions": "*", "php-edge-versions": "latest-stable" }
-  display_name: "OCSP tests - ${mongodb-versions}"
-  exclude_spec:
-    # OCSP is available from MongoDB 4.4+ (Debian 10 has MongoDB 4.4+)
-    - { "os": "debian10", "mongodb-versions": ["3.6", "4.0", "4.2"], "php-edge-versions": "latest-stable" }
-  tasks:
-    - name: ".ocsp"
-
-- matrix_name: "test-requireApiVersion"
-  matrix_spec: { "os": "debian11", "mongodb-versions": "*", "php-edge-versions": "latest-stable" }
-  display_name: "Versioned API - ${mongodb-versions}"
-  exclude_spec:
-    # Stable API is available from MongoDB 5.0+
-    - { "os": "debian11", "mongodb-versions": ["3.6", "4.0", "4.2", "4.4"], "php-edge-versions": "latest-stable" }
-  tasks:
-    - name: "test-requireApiVersion"
-
-- matrix_name: "test-loadBalanced"
-  matrix_spec: { "os": "debian11", "mongodb-versions": "*", "php-edge-versions": "latest-stable" }
-  display_name: "Load balanced - ${mongodb-versions}"
-  exclude_spec:
-    # Load balancer is available from MongoDB 5.0+
-    - { "os": "debian11", "mongodb-versions": ["3.6", "4.0", "4.2", "4.4"], "php-edge-versions": "latest-stable" }
-  tasks:
-    - name: "test-loadBalanced"
-
-- matrix_name: "test-csfle-skip_crypt_shared"
-  matrix_spec: { "os": "debian11", "mongodb-versions": "*", "php-edge-versions": "latest-stable" }
-  display_name: "CSFLE skip_crypt_shared - ${mongodb-versions}"
-  exclude_spec:
-    # CSFLE crypt_shared is available from MongoDB 6.0+
-    - { "os": "debian11", "mongodb-versions": ["3.6", "4.0", "4.2", "4.4", "5.0"], "php-edge-versions": "latest-stable" }
-  tasks:
-    - name: "test-skip_crypt_shared"
+include:
+  - filename: .evergreen/config/functions.yml
```

Comment:
Unfortunately, `include` is only allowed in top-level files, so we can't nest our includes (and include the various test files in `test-tasks.yml`).